yarn-project/aztec-rpc/src/database/tx_dao.ts

```diff
@@ -21,7 +21,7 @@ export class TxDao {
     /**
      * The sender's Aztec address.
      */
-    public readonly origin: AztecAddress,
+    public readonly origin: AztecAddress | undefined,
```

Comment:
Is the idea that, for a particular user, this db entry would only be truthy for transactions they _sent_? Presumably this will always be `undefined` for transactions that a user _receives_?

Reply:
Exactly! And this is actually not used anywhere. Don't think it's useful either. I'm removing it in another PR. Will open an issue for this :)